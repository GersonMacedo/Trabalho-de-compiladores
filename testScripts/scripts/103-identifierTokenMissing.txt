! -s SYNTACTIC
! should complain about missing identifier token
program ;
begin
end .