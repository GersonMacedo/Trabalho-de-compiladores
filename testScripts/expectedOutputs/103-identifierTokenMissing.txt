
Performing up to syntactic analysis at testScripts/scripts/103-identifierTokenMissing.txt
Parser::Expected '<identifier>' but ';' was found at line 3 column 9
